src/main/kotlin/io/github/lagersystembackend/attribute/AttributeRepository.kt

```diff
@@ -0,0 +1,11 @@
+package io.github.lagersystembackend.attribute
+
+interface AttributeRepository {
+    fun createOrUpdateAttribute(key: String, attribute: Attribute, id: String): Attribute
```

Comment:
Why do attributes exist without a binding productId?
Do we want to store attributes without any relations to a specific object?

Reply:
Do u mean why I made a AttributeRepository Interface and a ProductAttributeRepository? That's because maybe we want to add attributes to spaces or storages later.